src/reducer/dataReducer.jsx

```diff
@@ -0,0 +1,23 @@
+const initialReducerData = {
+  categoriesList: null,
+  productsList: null,
+  status: null,
+  error: null,
+};
+
+const dataReducer = (state, action) => {
```

Comment:
Correct file name in the PR description. I think, it is mis-spelled. And also, as in this file we are not using any `html` tag. So I would say to name it using `.js` extension and not `.jsx`

It contains only pure javascript thing.